docs/modules/operation/pages/flows/sentinel/message-broker/activemq.adoc

```diff
@@ -30,10 +30,10 @@ config:property-set broker-url failover:tcp://my-activemq-ip:61616<4>
 config:update
 ----
 
-<1> A location string is required and is used only for tracing
-<2> Unique identifier is used as service name only for tracing
-<3> Base URL for the web UI which provides the REST endpoints
-<4> URL which points to ActiveMQ broker.
+<1> A location string is required and used only for tracing.
+<2> Unique identifier used as service name only for tracing.
```

Comment:
```suggestion
```suggestion
<1> A location string is used to assign the Sentinel to a monitoring location with Minions.
<2> Unique identifier used as a node label for monitoring the Sentinel instance within {page-component-title}.
```
```